### DISCUSSION TITLE
Restart with Reason "Exception" every one to two days

### DISCUSSION THREAD

**Discussion by ModuloFS** [arendst/Tasmota#21065] [Support request]
### PROBLEM DESCRIPTION
Every one or two days the devices restarts with the restart reason "Exception". Every 10 to 20 exception restarts the device gets in a unknown state and is not reachable over wifi anymore. Just a Power Cycle brings the device back to life.

This is my template configuration:
{"NAME":"ESE86-03AJAI","GPIO":[544,227,289,34,226,33,0,0,32,224,290,225,288,0],"FLAG":0,"BASE":18}
https://templates.blakadder.com/SCS86-03AJAI.html

In this device a ESP8266MOD-12F is populated.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): https://templates.blakadder.com/SCS86-03AJAI.html with ESP8266MOD-12F
- [ ] Tasmota binary firmware version number used: 13.4.0
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: tasmota ota flasher
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
20:18:20.829 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Exception","Uptime":"0T03:11:26","StartupUTC":"2024-03-29T16:06:54","Sleep":50,"CfgHolder":4617,"BootCount":15,"BCResetTime":"2024-03-16T11:44:18","SaveCount":73,"SaveAddress":"F7000"}}
20:18:20.834 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS2 = {"StatusFWR":{"Version":"13.4.0(tasmota)","BuildDateTime":"2024-02-14T16:13:56","Boot":31,"Core":"2_7_6","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"429/699"}}
20:18:20.840 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":4,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Modulo-RNET",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["0000C009","2805C80001000600003C5A0A192800000000","40000080","00006000","00000000","00000000"]}}
20:18:20.855 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS4 = {"StatusMEM":{"ProgramSize":646,"Free":356,"Heap":21,"ProgramFlashSize":1024,"FlashSize":4096,"FlashChipId":"1640EF","FlashFrequency":40,"FlashMode":"DOUT","Features":["0809","8F9AC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","54000020","00000080","00000000"],"Drivers":"1,2,!3,!4,!5,!6,7,!8,9,10,12,!16,!18,!19,!20,!21,!22,!24,26,27,29,!30,!35,!37,!45,62,!68","Sensors":"1,2,3,4,5,6","I2CDriver":"7"}}
20:18:20.867 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS5 = {"StatusNET":{"Hostname":"rollo-wohnzimmer-west-rechts-55","IPAddress":"192.168.0.68","Gateway":"192.168.0.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.0.15","DNSServer2":"0.0.0.0","Mac":"2C:3A:E8:08:15:BA","Webserver":2,"HTTP_API":0,"WifiConfig":4,"WifiPower":17.0}}
20:18:20.872 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.0.15","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_0815BA","MqttUser":"DVES_USER","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
20:18:20.880 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS7 = {"StatusTIM":{"UTC":"2024-03-29T19:18:20Z","Local":"2024-03-29T20:18:20","StartDST":"2024-03-31T02:00:00","EndDST":"2024-10-27T03:00:00","Timezone":"+01:00","Sunrise":"06:32","Sunset":"19:16"}}
20:18:20.887 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS10 = {"StatusSNS":{"Time":"2024-03-29T20:18:20","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}}
20:18:20.894 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS11 = {"StatusSTS":{"Time":"2024-03-29T20:18:20","Uptime":"0T03:11:26","UptimeSec":11486,"Heap":21,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","Wifi":{"AP":1,"SSId":"Modulo-RNET","BSSId":"34:31:C4:35:45:F2","Channel":11,"Mode":"11n","RSSI":58,"Signal":-71,"LinkCount":1,"Downtime":"0T00:00:04"}}}
20:18:20.909 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS12 = {"StatusSTK":{"Exception":0,"Reason":"Exception","EPC":["3c22050f","00000000","00000000"],"EXCVADDR":"00000000","DEPC":"00000000","CallChain":["40100b38","40240000","40105d6a","40105d4c","4026b63c","400005e1","4026b63c","40283c93","40283c6c","4027fa75","40284fbd","402784f0","402762c5","40277f41","4026b501","40274dbb","4027467b","40252530","40000f49","40000f49","40000e19","40105b65","40105b6b","4010000d","40273560","40273511","40100b38","40106305","4000050c","4000e26b","4000d9f1"]}}
20:18:20.917 MQT: stat/rollo-wohnzimmer-west-rechts/STATUS13 = {"StatusSHT":{"SHT0":{"Relay1":1,"Relay2":2,"Open":570,"Close":570,"50perc":50,"Delay":0,"Opt":"0000","Calib":[300,500,700,900,1000],"Mode":"1"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Wait...

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Device should not do restarts and bricks itself every 10 days.

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Looks like a power supply issue. The wifi signal is weak too. A bad combination. Bad wifi means more current needed. So a border line power supply will gets in real problems.

**Reply by ModuloFS** [reply]
Ok, i will check the supply. But if the powersupply is bad, why i do not get the restart reason "Power On"? Is there no bronwout reset implemented in the MCU?

**Reply by Jason2866** [reply]
The usual behaviour from esp8266 when having power issues is having exceptions. Never seen brownout from esp8266. If the function is there it doesn't work.

**Reply by sfromis** [reply]
The ESP8266 doesn't know when it is going to :brown_circle:
